third-party/Makefile

```diff
@@ -62,6 +62,23 @@ clobber: FORCE
 
 depend:
 
+# Third-party dependencies. If any of these files (3rd party source dirs or our
+# Makefiles) change, then rebuild the 3rd party lib. This allows us to
+# automatically rebuild on 3rd party version upgrades or if we specify
+# different option in the Makefile.
+
+SRC_FILES = -iname '*.cc'  -o -iname '*.cp'  -o -iname '*.cxx' -o -iname '*.cpp' -o  -iname '*.CPP' -o -iname '*.c++' -o  -iname '*.hh'  -o -iname '*.hp' -o -iname '*.hxx' -o -iname '*.hpp' -o -iname '*.HPP' -o -iname '*.h++' -o -iname '*.tcc' -o -iname '*.c'   -o -iname '*.h'
```

Comment:
`SRC_FILES` is a pretty general name and I can imagine some poor naive soul using it for another purpose in such a way that it conflicts here.  Maybe something more descriptive, like `REBUILD_DEP_FILENAMES`?